frontend/src/screens/Card.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useRef, memo } from 'react'
```

Comment:
slay, i would just move it from screens to components (u can delete the card.js that's on components)